src/app/static/src/app/generated.d.ts

```diff
@@ -808,6 +808,21 @@ export interface SelectControl {
     }[];
   }[];
 }
+export interface MultiselectControl {
+  name: string;
+  label?: string;
+  type: "multiselect";
+  required: boolean;
+  value?: string[] | string;
+  helpText?: string;
+  options?: {
+    id: string;
+    label: string;
+    children?: {
+      [k: string]: any;
+    }[];
+  }[];
+}
```

Comment:
Not sure what this is about?

Reply:
Hmm so I regenerated the types because I thought it would need it to work with schema changes in the hintr PR. But I guess not because looks like no changes related to that. This is a change I added a couple of months ago see https://github.com/mrc-ide/hintr/pull/405/files#diff-6bd8a7b93ff7c312277151e42ef21432985c55b46dc4c46d27046e3bef94246fR33, but we made some hint changes related to this https://github.com/mrc-ide/hint/pull/747 maybe just forgot to regenerate types after that? Any ideas if that is going to cause problems?

Reply:
Added some tests for this now!